index.ts

```diff
@@ -59,12 +59,41 @@ const plugin: Plugin<ReplicatorMetaInput> = {
         }
 
         if (batch.length > 0) {
-            await fetch(`https://${config.host.replace(/\/$/, "")}/e`, {
+            const batchDescription = `${batch.length} event${batch.length > 1 ? 's' : ''}`
+            await fetch(`https://${config.host.replace(/\/$/, '')}/e`, {
```

Comment:
Can we add a TODO to add a fetch timeout, if you think it's relevant?

Reply:
Adding the timeout is a simple as `signal: timeoutSignal(20000)`, but testing it might be a rabbit hole I'd prefer to keep for later ^^
Adding a TODO